webroot/results/admin/persons_finish_unfinished_ACTION.php

```diff
@@ -125,6 +141,30 @@ function insertPerson( $newName, $newCountry, $newId ) {
   
   #--- Execute the command.
   dbCommand( $command );
+
+  if ( $oldCountry <> $newCountry || $oldName <> $newName){
+
+    #--- Guess countries iso2
+    $newCountryIso2 = getIso2FromCountryId( $newCountry );
+    if ( $newCountryIso2 ){
+      if ( $oldCountry == $newCountry ){
+        $oldCountryIso2 = $newCountryIso2;
+      } else {
+        $oldCountryIso2 = getIso2FromCountryId( $oldCountry );
+      }
+      if ( $oldCountryIso2 ){
+        #--- Build the command.
+        $command = "
+          UPDATE InboxPersons SET name='$newName', countryId='$newCountryIso2'
```

Comment:
Yeah, sure!  I'm afraid there is much to improve in this process when it is ported to Rails.